app/models/Group.js

```diff
@@ -66,6 +67,11 @@ module.exports = function(Sequelize, DataTypes) {
       }
     },
 
+    hostFeePercent: {
+      type: DataTypes.FLOAT,
+      defaultValue: constants.HOST_FEE_PERCENT
```

Comment:
I don't disagree - it's cleaner if computers are the only ones dealing with it. We'll be manually editing it for the time being. One comparison point is that stripe also calls its field: application_fee_percent and it's an Int between 0 and 100.